src/main/scala/latis/input/HapiAdapter.scala

```diff
@@ -39,7 +41,7 @@ abstract class HapiAdapter(model: DataType, config: HapiAdapter.Config) extends
   private var baseUriString: String = _
 
   /** Defines the format of time strings used by the HAPI API. */
-  val timeFormat: TimeFormat = TimeFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+  val timeFormat: TimeFormat = TimeFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") //TODO: consider relaxing this
```

Comment:
Where are we using this now that we changed the time coverage parsing below? Maybe just for formatting output?

Reply:
It seems the only other place it's used is lower down in this file, to build the query:

```scala
// Builds the query
Seq(
  s"id=${config.id}", // Add dataset ID
  s"time.min=${timeFormat.format(startTime)}",
  s"time.max=${timeFormat.format(stopTime)}",
  params,
  s"format=$datasetFormat" // Add dataset format
).mkString("&")
 ```

Reply:
Should we make this private?

Reply:
I'm all for it. Doesn't need to be used elsewhere, and I could remove the TODO with it private.

Reply:
Made it private.

Reply:
Note that the latest latis3 requires `TimeFormat.fromExpression` now. This also happens to be the same as our `TimeFormat.Iso`. Presumably it would be safe to use that.

Reply:
Sorry, what are you talking about @dlindhol? Maybe I missed something, but I don't know what `TimeFormat.fromExpression` is, and when I type `TimeFormat.` in IntelliJ, `fromExpression` isn't in the list of autofill suggestions.

I see that `TimeFormat.Iso` is the same as the string we have here, but are we 100% certain we'll never change that value? It seems more safe to me to hardcode exactly the ISO string defined in the HAPI spec like this, in case we ever change our `TimeFormat.Iso` value.

Reply:
You'll need to update the latis3 version to see the change. I tried to make the time package a bit more type safe recently.

I agree that since HAPI specifies a format, we should explicitly use that here where we are building a HAPI server query. Presumably a HAPI server would parse any ISO time we generate (and we wouldn't change such things), but....

Note that parsing time values that we get from a HAPI server is a separable issue (as we ran into with the time coverage info). Our `TimeFormat.fromIsoValue` (used by `parseIso`) should work for any ISO they consider valid (https://github.com/hapi-server/data-specification/blob/master/hapi-dev/HAPI-data-access-spec-dev.md#376-representation-of-time).

Reply:
I gotcha. I updated the build.sbt here to point at the latest commit then used `TimeFormat.fromExpression`. The code on this line is now identical to `TimeFormat.Iso`, but regardless I'll sleep better knowing we were explicit.